src/test/java/com/hubspot/jinjava/lib/tag/eager/EagerSetTagTest.java

```diff
@@ -122,4 +122,16 @@ public void itThrowsAndDefersVarWhenValContainsDeferred() {
   public void itThrowsAndDefersMultiVarWhenValContainsDeferred() {
     // Deferred values are handled differently. Test does not apply.
   }
+
+  @Test
+  @Ignore
+  public void itSetsBlock() {
+    // not implemented yet
+  }
+
+  @Test
+  @Ignore
+  public void itSetsBlockWithFilter() {
+    // not implemented yet
+  }
```

Comment:
It is complex to properly support this with eager execution so it's in a separate PR https://github.com/HubSpot/jinjava/pull/699